build/helpers/config.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * @file Provides simple user configuration options.
+ */
+
+import loconfig from '../../loconfig.json' assert { type: 'json' };
+import { merge } from '../utils/index.js';
+
+try {
+    const usrconfig = await import('../../loconfig.local.json', {
+        assert: { type: "json" },
+    });
+    usrconfig = usrconfig.default;
+
+    merge(loconfig, usrconfig);
+} catch (err) {
+    // do nothing
+}
+
+export default loconfig;
+
+export {
+    loconfig,
```

Comment:
`loconfig` is already exported. And merge export is missing.
```suggestion
    merge,
```

Reply:
I've resolved this by having `watch.js` import `merge` from `utils`.